website/content/en/rules/gke/BP/2021_002.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "gke/BP/2021_002"
+linkTitle: "BP/2021_002"
+weight: 1
+type: docs
+description: >
+  Google Groups for RBAC enabled.
+---
+
+**Product**: [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine)\
+**Rule class**: BP - Best practice, opinionated recommendation
+
+### Description
+
+IAM and Kubernetes RBAC work together to help manage access to your cluster.
```

Comment:
We should whenever possible reference external documentation instead of adding it here (and the text seems to be copied from the public docs). For rule pages I would focus on the following:

- Why was the rule triggered
- Short description about why this is a recommended best-practice (i.e. the same description that you have in the rule implementation "Enable Google Groups for RBAC so cluster...".
- Hints about how to solve the issue (possibly also just pointing to public docs)
- Links to public documentation for further information.

Reply:
This document still contains too much non-original content in my opinion. I would remove even more text.

Reply:
Reduced text.